include/circt/Dialect/SV/SV.td

```diff
@@ -43,6 +43,7 @@ def VendorExtension : NativeOpTrait<"VendorExtension"> {
 }
 
 include "circt/Dialect/HW/HWTypes.td"
+include "circt/Dialect/HW/HWAttributes.td"
```

Comment:
Something seems funky if this is needed here. Is this still needed given the changes to `SVTypeDecl.td`?